src/test/java/guitests/PersonEditGuiTest.java

```diff
@@ -39,60 +38,56 @@ public void editPerson_usingAccelerator() {
         //Get a reference to the card displaying Alice's details
         PersonCardHandle alicePersonCard = personListPanel.getPersonCardHandle(td.alice);
 
-        //Edit Alice to change to new values
-        personListPanel.clickOnPerson(td.alice);
-        EditPersonDialogHandle editPersonDialog = personListPanel.use_PERSON_EDIT_ACCELERATOR();
-        editPersonDialog.enterNewValues(newAlice);
-        editPersonDialog.pressEnter();
+        personListPanel.rightClickOnPerson(td.alice);
+        EditPersonDialogHandle editPersonDialog =
+                personListPanel.clickOnContextMenu(PersonListPanelHandle.ContextMenuChoice.EDIT);
```

Comment:
It's not correct for context menu click to return a handle to edit dialog. A context menu click may not always lead to an edit op.